tests/tfhe_rust_bool/end_to_end_fpga/src/main.rs

```diff
@@ -72,10 +72,11 @@ fn main() {
     let ct_1 = encrypt(flags.input1.into(), &client_key);
     let ct_2 = encrypt(flags.input2.into(), &client_key);
 
-    let ct_1= ct_1.iter().collect();
-    let ct_2= ct_2.iter().collect();
-
+    let t = Instant::now();
     let result = fn_under_test::fn_under_test(&server_key, &ct_1, &ct_2);
+    let run = t.elapsed().as_millis();
```

Comment:
```suggestion
    // let t = Instant::now();
    let result = fn_under_test::fn_under_test(&server_key, &ct_1, &ct_2);
    // let run = t.elapsed().as_millis();
```

Reply:
Or delete them

Reply:
I have added an additional comment
I know it is not the right thing to do, but it is for obtaining quick timing results when run on fpga (turn the printing off for the user/sys timings)